tests/integration/docker_utils/test_docker.py

```diff
@@ -118,6 +118,16 @@ def _create_network(network_name: str):
 
 
 class TestDockerClient:
+    def test_get_system_info(self, docker_client: ContainerClient):
+        info = docker_client.get_system_info()
+        assert "ID" in info
+        assert "OperatingSystem" in info
+        assert "Architecture" in info
+
+    def test_get_system_id(self, docker_client: ContainerClient):
+        assert len(docker_client.get_system_id()) > 1
+        assert docker_client.get_system_id() == docker_client.get_system_id()
```

Comment:
This part will fail on podman, as the id is regenerated every time. Might be problematic for our podman tests, once ubuntu fixes it's podman networking packages :)

Reply:
that's very good to know!